### PR TITLE
Add tests for Assert.Fail().

### PR DESCRIPTION
**Pull request by josteink**
This PR depends on https://github.com/xunit/assert.xunit/pull/34 being merged before compiling and running cleanly.

When both are merged, this closes https://github.com/xunit/xunit/issues/2027.

### CODE REVIEW

File: test/test.xunit.assert/Asserts/FailAssertsTests.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Sdk;
+
+namespace test.xunit.assert.Asserts
```
Comment (Kerl1310) [inline test/test.xunit.assert/Asserts/FailAssertsTests.cs]: Looking at the other tests, this isn't required

Comment (josteink) [reply test/test.xunit.assert/Asserts/FailAssertsTests.cs]: Fair enough. Namespace removed.

**Comment by josteink** [comment]
@bradwilson: same as other thread

**Comment by josteink** [comment]
If there’s no interest for these PRs, I’ll just close them.

**Pull request closed without merging**